src/main/scala/com/softwaremill/realworld/articles/core/ArticlesRepository.scala

```diff
@@ -162,16 +158,35 @@ class ArticlesRepository(quill: Quill.Sqlite[SnakeCase]):
           _.updatedAt -> lift(now),
           _.authorId -> lift(userId)
         )
-        .returning(_.articleId)
-    )
-      .pipe(mapUniqueConstraintViolationError)
-  }
+        .returning[Int](_.articleId)
+    }
 
-  def deleteArticle(articleId: Int): Task[Long] =
-    run(queryArticle.filter(_.articleId == lift(articleId)).delete)
+    def addTags(tags: List[String], articleId: Int) =
```

Comment:
**suggestion:** Can you add explicit return type to this and other public `defs`?